examples/3d/auto_exposure.rs

```diff
@@ -101,11 +101,11 @@ fn setup(
         }
     }
 
-    commands.insert_resource(AmbientLight {
-        color: Color::WHITE,
-        brightness: 0.0,
-        ..default()
-    });
+    #[expect(
+        deprecated,
+        reason = "Once AmbientLight is removed, the resource can be removed"
+    )]
+    commands.insert_resource(AmbientLight::NONE);
```

Comment:
Example code shouldn't use deprecated things, they need to be updated to use the new component.